FactoryWeather/App/VIP Home/Data source/HomeDataSource.swift

```diff
@@ -9,7 +9,7 @@
 import UIKit
 import CoreLocation
 
-struct HomeViewModel {
+struct HomeDataSource {
```

Comment:
I would actually use `HomeViewModel` here, as it's indeed all of the `View` data ready for display. Or am I wrong with that assumption?

```suggestion
struct HomeViewModel {
```

Additionally, we should have a `typealias ViewModel = (x: ?, y: ?, ...)` for each of our `views`. Then, the `Presenter` passes-on all of the necessary `ViewModel` types that the `ViewController` then uses to update it's views.

Reply:
I removed `HomeDataSource` and moved data mapping to `Presenter`. Also i added typealias `ViewModel` to `HomeContentView`.